components/screens/MenuScreen.tsx

```diff
@@ -128,7 +128,7 @@ export const MenuScreen = (queryCache: QueryCache, avalancheCenterId: AvalancheC
                 </Body>
               </Card>
               <View py={16} px={32}>
-                <Button buttonStyle="primary" onPress={() => void Linking.openURL(`mailto:charlotte@nwac.us?subject=I%20have%20thoughts`)}>
+                <Button buttonStyle="primary" onPress={() => void sendMail({to: 'charlotte@nwac.us', subject: 'NWAC app feedback', logger})}>
```

Comment:
nit: can we send this to `developer@nwac.us`? No need to bottle-neck with Charlotte if we don't need to

Reply:
I'll do `developer+app-feedback@nwac.us` so we can filter it more easily